examples/api/lib/material/icon_button/icon_button.3.dart

```diff
@@ -36,153 +36,105 @@ class DemoIconToggleButtons extends StatefulWidget {
-            children: <Widget>[
-              // Filled IconButton
-              DemoIconToggleButton(isEnabled: true, getDefaultStyle: enabledFilledButtonStyle,),
-              SizedBox(width: 10),
-              DemoIconToggleButton(isEnabled: false, getDefaultStyle: disabledFilledButtonStyle,)
-            ]
-          ),
-          Row(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: <Widget>[
-              // Filled Tonal IconButton
-              DemoIconToggleButton(isEnabled: true, getDefaultStyle: enabledFilledTonalButtonStyle,),
-              SizedBox(width: 10),
-              DemoIconToggleButton(isEnabled: false, getDefaultStyle: disabledFilledTonalButtonStyle,),
-            ]
-          ),
-          Row(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: <Widget>[
-              // Outlined IconButton
-              DemoIconToggleButton(isEnabled: true, getDefaultStyle: enabledOutlinedButtonStyle,),
-              SizedBox(width: 10),
-              DemoIconToggleButton(isEnabled: false, getDefaultStyle: disabledOutlinedButtonStyle,),
-            ]
-          ),
-        ]
-      ),
-    );
-  }
-}
-
-class DemoIconToggleButton extends StatefulWidget {
-  const DemoIconToggleButton({required this.isEnabled, this.getDefaultStyle, super.key});
-
-  final bool isEnabled;
-  final ButtonStyle? Function(bool, ColorScheme)? getDefaultStyle;
-
-  @override
-  State<DemoIconToggleButton> createState() => _DemoIconToggleButtonState();
-}
-
-class _DemoIconToggleButtonState extends State<DemoIconToggleButton> {
-  bool selected = false;
+  bool standardSelected = false;
+  bool filledSelected = false;
+  bool tonalSelected = false;
+  bool outlinedSelected = false;
 
   @override
   Widget build(BuildContext context) {
-    final ColorScheme colors = Theme.of(context).colorScheme;
-    final VoidCallback? onPressed = widget.isEnabled
-      ? () {
-        setState(() {
-          selected = !selected;
-        });
-      }
-      : null;
-    ButtonStyle? style;
-    if (widget.getDefaultStyle != null) {
-      style = widget.getDefaultStyle!(selected, colors);
-    }
-
-    return IconButton(
-      isSelected: selected,
-      icon: const Icon(Icons.settings_outlined),
-      selectedIcon: const Icon(Icons.settings),
-      onPressed: onPressed,
-      style: style,
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: <Widget>[
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            IconButton(
+              isSelected: standardSelected,
+              icon: const Icon(Icons.settings_outlined),
+              selectedIcon: const Icon(Icons.settings),
+              onPressed: () {
+                setState(() {
+                  standardSelected = !standardSelected;
+                });
+              },
+            ),
+            const SizedBox(width: 10,),
```

Comment:
In cases like this we don't usually include the trailing comma:
```suggestion
            const SizedBox(width: 10),
```

Here and elsewhere.